kani/engines/ctransformers/llama2.py

```diff
@@ -0,0 +1,64 @@
+import warnings
+
+from kani.ai_function import AIFunction
+from kani.exceptions import MissingModelDependencies
+from kani.models import ChatMessage, ChatRole
+from .base import CTransformersEngine
+
+B_INST, E_INST = "[INST]", "[/INST]"
+B_SYS, E_SYS = "<<SYS>>\n", "\n<</SYS>>\n\n"
+
+class LlamaCTransformersEngine(CTransformersEngine):
+    """Implementation of LLaMA v2 using huggingface transformers.
+
+    You may also use the 13b, 70b, or other LLaMA models that use the LLaMA prompt by passing the HuggingFace model
+    ID to the initializer.
+
+    Model IDs:
+
+    - ``TheBloke/Llama-2-7B-Chat-GGML``
+    - ``TheBloke/Llama-2-13B-chat-GGML``
+
+    .. code-block:: python
+
+        engine = LlamaCTransformersEngine("TheBloke/Llama-2-7B-Chat-GGML", model_file="llama-2-7b-chat.ggmlv3.q5_K_M.bin")
+        ai = Kani(engine)
+    """
+
+    def __init__(self, model_id: str = "TheBloke/Llama-2-7B-Chat-GGML", model_file: str = "llama-2-7b-chat.ggmlv3.q5_K_M.bin", *args, **kwargs):
```

Comment:
Could you add more info about where to obtain the model file to the docs? (Or does it automatically download the right file? I've never used ctransformers)